lib/all_your_base_cases.rb

```diff
@@ -57,16 +57,17 @@ def error_message
 
 class AllYourBaseCase::PreProcessor
   class << self
+    attr_reader :row
+
     def call(row)
       @row = row
 
       row.merge('expected' => expected_value)
     end
 
+    private :row
     private
 
```

Comment:
I don't think it needs an explicit trigger. Next time that code gets edited, whoever modifies it will think it looks wrong,  (unjustly) curse @tommyschaefer for being an idiot :stuck_out_tongue_winking_eye: and fix it.